internal/characters/lauma/dash.go

```diff
@@ -0,0 +1,15 @@
+package lauma
+
+import (
+	"errors"
+
+	"github.com/genshinsim/gcsim/pkg/core/action"
+)
+
+func (c *char) Dash(p map[string]int) (action.Info, error) {
+	if c.StatusIsActive(deerStatusKey) {
+		return action.Info{}, errors.New("dash called in deer state")
```

Comment:
`return action.Info{}, errors.New("cannot use dash in lauma spirit envoy state")`